Makefile

```diff
@@ -34,13 +35,15 @@ estimate-model:
 	python3 Tools/parametric_model/generate_parametric_model.py \
 	--config ${config} \
 	--data_selection ${data_selection} \
+	--selection_variable ${selection_variable} \
```

Comment:
Could we move this from the command line to the config file?

Reply:
yes, absolutely. If we have this parameter in the configuration then anyways, I will probably also add a possibility to configure, which activations should be used for identification. This was difficult before, because passing lists to the program through makefile seems to be a bit painful - using the config files, however, I think it shouldn't be a problem (and I already have the changes locally)

Reply:
@Jaeyoung-Lim The user can now specify "setpoint" as the data_selection method and then the settings from the config file will be used where the variable and the activations can be specified. This should be a relatively good starting point now for further developments to integrate it in the UI as at least no code changes are required anymore ^^